### PR TITLE
Update route policy params to Bgp nbr AF RM

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### SUMMARY

fixes: https://github.com/ansible-collections/cisco.iosxr/issues/151 and https://github.com/ansible-collections/cisco.iosxr/issues/153
This change will go as BUGFIX as route_policy option has never worked earlier. 
So this won't be breaking change.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

iosxr_bgp_neighbor_address_family

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by ashwini-mhatre** [comment]
recheck

File: plugins/module_utils/network/iosxr/rm_templates/bgp_neighbor_address_family.py [hunk 1]
```diff
@@ -673,6 +673,66 @@ def __init__(self, lines=None):
                 }
             }
         },
+        {
+            "name": "route_policies.inbound",
```
Comment (NilashishC) [inline plugins/module_utils/network/iosxr/rm_templates/bgp_neighbor_address_family.py:677]: Will `route_policy` still be rendered in facts?

Comment (ashwini-mhatre) [reply plugins/module_utils/network/iosxr/rm_templates/bgp_neighbor_address_family.py:677]: no it won't as in running config there is always "route-policy route-policy-name in/out" this line will present.

Comment (NilashishC) [reply plugins/module_utils/network/iosxr/rm_templates/bgp_neighbor_address_family.py:677]: Then we can remove `route_policy` parser? Also, what happens if someone sets `route_policy` option in task?

**Review by NilashishC — APPROVED** [review]

**Comment by ashwini-mhatre** [comment]
recheck

**Pull request merged**